contracts/access/AccessControl.sol

```diff
@@ -42,18 +40,11 @@ import "../utils/Context.sol";
  * accounts that have been granted it.
  */
 abstract contract AccessControl is Context {
-    using EnumerableSet for EnumerableSet.AddressSet;
-    using Address for address;
-
-    struct RoleData {
-        EnumerableSet.AddressSet members;
-        bytes32 adminRole;
-    }
-
-    mapping (bytes32 => RoleData) private _roles;
```

Comment:
Was there a reason to remove the `RoleData` struct and replace this mapping with the two separate mappings below? I would've kept it, just changing `members` into a mapping.